microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,119 @@ description: Version Models used for Microsoft Edge WebView2
 title: Understand WebView2 SDK versions
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 05/06/2021
+ms.date: 07/13/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
 ---
-# Understand WebView2 SDK versions  
+# Understand WebView2 SDK versions
 
 New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.  
 
-## Release and prerelease package  
+When you develop and distribute a WebView2 app, the client must at least have a version of the Runtime that can use the core APIs and create a WebView2.  The client ideally should have a version of the Runtime that supports all of the latest APIs that are in the SDK version you use to develop the app.  You must coordinate the WebView2 SDK version that you use for development, and the WebView2 Runtime version that's installed on client machines.
 
-The WebView2 NuGet package contains both a release and pre-release package.  
+*  Core API functionality requires that the client has WebView2 Runtime version `86.0.616.0` or newer.
```

Comment:
I would avoid using the word Core here and in other places in this doc, as it may confuse folks to think this refers to anything in the Core namespace (https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core?view=webview2-dotnet-1.0.864.35), which is not true.

I think what we want to convey in this doc is that `86.0.616.0` is the necessary (but not sufficient) min version for any WebView2 API. WebView2 APIs are forward-compatible and can work on a Runtime that has a higher build number than the SDK build number in which the API is introduced. E.g. API_x is introduced in SDK 1.0.900.0, so it would work in runtime 94.0.1000.0 but not runtime 90.0.700.0. I'm making numbers up here but hopefully you get the point. `86.0.616.0` is kinda special because this is the version we GA with, so any WebView2 API that we GA'ed with would work on 616 and onwards. What you mentioned as Core is basically the set of API we GA'ed with. 

When developers are using an SDK that has lower or equal build number than runtime, every API they have access to works. When developers are using an SDK that has a greater build number than runtime, some of the APIs' implementation are not yet available in the runtime therefore cannot work. Since evergreen developers can't really control the runtime version (it pausing updates or offline), it's a good practice to feature-detect (QI/try-catch) when using relatively new APIs.